plugins/outputs/stomp/stomp.go

```diff
@@ -0,0 +1,135 @@
+package stomp
+
+import (
+	ts "crypto/tls"
+	"net"
+
+	"github.com/go-stomp/stomp"
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/common/tls"
+	"github.com/influxdata/telegraf/plugins/outputs"
+	"github.com/influxdata/telegraf/plugins/serializers"
```

Comment:
Maybe using this aliases would be better
```suggestion
	"crypto/tls"
	"net"

	"github.com/go-stomp/stomp"
	
	"github.com/influxdata/telegraf"
	commontls "github.com/influxdata/telegraf/plugins/common/tls"
	"github.com/influxdata/telegraf/plugins/outputs"
	"github.com/influxdata/telegraf/plugins/serializers"
```

Reply:
done